Tests/private_build/configure_and_test_integration_instances_private.py

```diff
@@ -10,10 +10,8 @@
 
 from Tests.scripts.utils.log_util import install_logging
 from demisto_sdk.commands.common.tools import find_type
-from Tests.configure_and_test_integration_instances import Build, configure_servers_and_restart, \
-    get_tests, \
-    get_changed_integrations, configure_server_instances, instance_testing, disable_instances, \
-    report_tests_status, nightly_install_packs, test_files, \
+from Tests.configure_and_test_integration_instances import XSOARBuild, Build, \
```

Comment:
did we test the changes here against content-private build?

Reply:
It used to work long time ago, after my changes. Private build is failing for long time... 
I didn't added XSIAM machines to the private build.